### PR TITLE
Added support for django 1.11

### PR DESCRIPTION
**Pull request by ahsan-ul-haq**
LEARNER-1182
Added support for Django 1.11 and Python 3.6.

### CODE REVIEW

File: edx_rest_framework_extensions/__init__.py [hunk 1]
```diff
@@ -1,3 +1,3 @@
 """ edx Django REST Framework extensions. """
 
-__version__ = '1.2.2'  # pragma: no cover
+__version__ = '1.3.0'  # pragma: no cover
```
Comment (clintonb) [inline edx_rest_framework_extensions/__init__.py]: This is not worthy of a minor version bump. 1.2.3

File: tox.ini [hunk 2]
```diff
@@ -10,6 +10,7 @@ deps =
     django18: Django>=1.8,<1.9
     django19: Django>=1.9,<1.10
     django110: Django>=1.10,<1.11
```
Comment (clintonb) [inline tox.ini]: This is probably why the tests are failing.

File: tox.ini [hunk 3]
```diff
@@ -10,7 +10,7 @@ deps =
...
-    django110: Django>=1.11,<1.12
+    django111: Django>=1.11,<1.12
```
Comment (clintonb) [inline tox.ini]: There won't be a Django 1.12. Next version is 2.0.

**Comment by ahsan-ul-haq** [comment]
@clintonb made the suggested changes

**Comment by ahsan-ul-haq** [comment]
@clintonb made the change

**Pull request merged**